backend/src/seed/factories/organizations.js

```diff
@@ -1,21 +0,0 @@
-import faker from 'faker'
-import uuid from 'uuid/v4'
-
-export default function create(params) {
-  const {
-    id = uuid(),
-    name = faker.company.companyName(),
-    description = faker.company.catchPhrase(),
-  } = params
-
-  return {
-    mutation: `
-      mutation($id: ID!, $name: String!, $description: String!) {
-        CreateOrganization(id: $id, name: $name, description: $description) {
-          name
-        }
-      }
-    `,
-    variables: { id, name, description },
-  }
-}
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-03T11:49:13Z" title="Tuesday, September 3rd 2019, 1:49:13 pm +02:00">Sep 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,21 +0,0 @@
-import faker from 'faker'
-import uuid from 'uuid/v4'
-
-export default function create(params) {
-  const {
-    id = uuid(),
-    name = faker.company.companyName(),
-    description = faker.company.catchPhrase(),
-  } = params
-
-  return {
-    mutation: `
-      mutation($id: ID!, $name: String!, $description: String!) {
-        CreateOrganization(id: $id, name: $name, description: $description) {
-          name
-        }
-      }
-    `,
-    variables: { id, name, description },
-  }
-}
```


we'll add this back when we need it, right?
no point in spending time refactoring it now, if we don't plan on using it

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-03T13:44:38Z" title="Tuesday, September 3rd 2019, 3:44:38 pm +02:00">Sep 3, 2019</time>_

---


Yes.